.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 
 node_js:
+  - iojs
+  - '0.12'
```

Comment:
@mhegazy yes it will run all environments every time. Though some times not starting at the same time. All environments must pass to be considered a pass. 

Keep it to `iojs` and `node`?